assets/scss/utilities/_spacing.scss

```diff
@@ -56,4 +58,20 @@
 
 .has-reset-padding {
   padding: 0;
+}
+
+.has-gutter-padding {
+  padding: 0 $size-xs;
+  @include mq($from: bp-m, $until: bp-xl) {
+    padding: 0 $size-b;
+  }
+  @include mq($from: bp-xl) {
+    padding: 0
+  }
+}
+
+.has-temp-gutter-padding {
```

Comment:
I never understood what "temp" meant. My guess is Becca intended eventually to remove it. Do you know? If it's a mystery, we might consider renaming.